crates/containerd-shim-wasmtime/src/instance.rs

```diff
@@ -48,21 +86,64 @@ impl<T: WasiConfig> Default for WasmtimeEngine<T> {
     }
 }
 
-/// Data that contains both wasi_preview1 and wasi_preview2 contexts.
-pub struct WasiCtx {
```

Comment:
Notice that his struct is currently public so renaming should be considered an API breaking change.

Reply:
It's okay to have breaking changes - this crate is designed as to compile to a binary and also it's not reached 1.0 yet.